### ISSUE TITLE
Medical Cost Predictive Analysis

### ISSUE THREAD

**Issue by sitamgithub-MSIT** [abhisheks008/ML-Crate#483]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Medical Cost Predictive Analysis
:red_circle: **Aim** : Predict the medical insurance charge cost of individuals based on BMI, sex, smoker status, regions, and other features.
:red_circle: **Dataset** : [Kaggle Dataset](https://www.kaggle.com/datasets/mirichoi0218/insurance)
:red_circle: **Approach** : I will use 3–4 algorithms to implement the models and compare all the algorithms to find the best-fit algorithm for the model by checking the accuracy scores. Also, I will add an exploratory data analysis before creating model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Sitam Meur
- GitHub Profile Link : https://github.com/sitamgithub-MSIT
- Participant ID (If not, then put NA) :
- Approach for this Project : 
1. Import necessary libraries like numpy, pandas, scikit-learn, etc.
2. Performing the necessary pre-processing, encoding, scaling, etc.
3. Creating all the required functions. Probably some visualization utility functions and others.
4. Split the data into test and training datasets.
5. Train the model on linear regression, xg-boost, and other algorithms.
6. Checking the accuracy, like the r2 score and others.
At the end, label all the accuracy of all models in a table and do a comparative analysis between them.
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.) KWOC Participant

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Issue assigned to you. Make sure you merge the PR before the event deadline.
@sitamgithub-MSIT
